### ISSUE TITLE
Ability to publish and share a chat session

### ISSUE THREAD

**Issue by andrewm4894** [LAION-AI/Open-Assistant#2400]
Similar to how sharegpt works but just build it in to open assistant.

It's stilly people have to share screenshots and also means you often don't see full chat so can be misleading.

Ability to publish and easily share a chat session in some way would be cool. 

Or even just download as pdf (or something, maybe could add ability to save as a Google docs or something even) as a start if hosting too much hassle etc.

**Comment by Bzsmrf** [comment]
@andrewm4894 can i work on this i would love to help

**Comment by AbdBarho** [comment]
@Bzsmrf thank you, I think we have to think about this a little bit from a GDPR point of view, we might need to have an explicit "share" feature.

**Comment by Bzsmrf** [comment]
@AbdBarho I will be here . Just let me know when u decide how we are going about this

**Comment by andrewm4894** [comment]
I see the chats are saved and all have a UID looking url so guess they already there somewhere. So maybe the feature is more like a way to just make a chat public and publish it or something. 

Or an easier stepping stone could just be to download chat as pdf or something that user can then do whatever they want with.

**Comment by notmd** [comment]
Yeah, we can add an option for users to make it public or private. Or so warn them that any got the link will be able to view the hold conversation if they make it public
